gradle.properties

```diff
@@ -17,5 +17,5 @@
 ################################################################################
 org.gradle.caching=true
 org.gradle.parallel=true
-org.gradle.configureondemand=true
+#org.gradle.configureondemand=true
```

Comment:
What's the reason for this?

Reply:
I had some issues with JDK version not being applied to subprojects, so I disabled this beta feature.